go.mod

```diff
@@ -10,6 +10,8 @@ require (
 	github.com/google/go-github v17.0.0+incompatible
 	github.com/hashicorp/go-multierror v0.0.0-20171204182908-b7773ae21874
 	github.com/kr/pretty v0.1.0
+	github.com/onsi/ginkgo v1.7.0 // indirect
```

Comment:
This change seems unrelated?

Reply:
The tool chain did this change. It's becauee an earlier PR from John removed this since he's testing out go 1.13. Since I'm still on 1.12, that version of the compiler wants those two imports. When we switch everything to 1.13, then we'll refresh all the go.mod files.

Reply:
Then please put that into its own PR. It is unrelated to this change.

Reply:
Note that bazel avoids these sorts of problems, by declaring -- and using -- a particular go version.